app/views/shared/support_text_household.yml

```diff
@@ -4,10 +4,11 @@
 
 is_applying_coverage: "<b> Does this person need coverage? </b> <br> <br> If you need coverage, select ‘yes’. If you’re applying for coverage for someone else, but don’t need coverage for yourself, select ‘no’. If you don’t need or qualify for coverage, but someone in your family or that you include on your tax return does, you can apply on their behalf."
 us_citizen: "<b> Is this person a US citizen or US national? </b> <br> <br> You’re a US citizen if you were born in the US or certain US territories, had a parent who was a US citizen when you were born, or you became a US citizen through adoption or naturalization.<br> <br> A US national is a US citizen or anyone who owes permanent allegiance to the US."
-naturalized_citizen: "<b> Is this person a naturalized citizen? </b> <br> <br> A naturalized citizen is someone who wasn’t born a US citizen. A person can apply to be a naturalized citizen after being a permanent resident (green card holder) for a certain period of time, and if they meet certain other legal requirements."
+naturalized_citizen: "<b> Is this person a naturalized or derived citizen? </b> <br> <br> A naturalized citizen is someone who wasn’t born a US citizen. A person can apply to be a naturalized citizen after being a permanent resident (green card holder) for a certain period of time, and if they meet certain other legal requirements. <br> <br> <b>A derived citizen is: </b> <br> <br> <ul> <li>A person who derives U.S. citizenship through his or her relationship to a U.S. citizen (by operation of law).</li><li>A child who derives citizenship through the naturalization of the child's parents, through passage of certain laws, or through adoption by U.S. citizen parents.</li> <br> A person who acquires U.S. citizenship may have a Certificate of Citizenship (Form N-560 or N-561)."
```

Comment:
Put this in the Maine specific file `config/client_config/me/components/financial_assistance/app/views/financial_assistance/shared/support_text.yml`